### PR TITLE
add r-redlistr conda recipe

### PR DESCRIPTION
**Pull request by Marie59**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Marie59** [comment]
Hi @conda-forge/r ! This recipe is ready for review

File: recipes/r-redlistr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    - r-sp >=1.2_4
+
+test:
+  commands:
+    - $R -e "library('redlistr')"           # [not win]
+    - "\"%R%\" -e \"library('redlistr')\""  # [win]
+
+about:
+  home: https://github.com/red-list-ecosystem/redlistr
+  license: GPL-3
```
Comment (mfansler) [inline recipes/r-redlistr/meta.yaml:47]: Should be `GPL-3.0-or-later`. This is the SPDX interpretation of `License: GPL (>= 3)`.

File: recipes/r-redlistr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  summary: A toolbox created by members of the International Union for Conservation of Nature
+    (IUCN) Red List of Ecosystems Committee for Scientific Standards. Primarily, it
+    is a set of tools suitable for calculating the metrics required for making assessments
+    of species and ecosystems against the IUCN Red List of Threatened Species and the
+    IUCN Red List of Ecosystems categories and criteria. See the IUCN website for detailed
+    guidelines, the criteria, publications and other information.
+  license_family: GPL3
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
```
Comment (mfansler) [inline recipes/r-redlistr/meta.yaml:56]: Authors also have a separate [`LICENSE` file](https://github.com/red-list-ecosystem/redlistr/blob/master/LICENSE) in the package, which should also be included. The DESCRIPTION file shows `License: GPL (>= 3) | file LICENSE`, indicating the authors are intent on its inclusion.

File: recipes/r-redlistr/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
+    - marie59
+
+# Package: redlistr
```
Comment (mfansler) [inline recipes/r-redlistr/meta.yaml:63]: The commented out copy of the DESCRIPTION file can be removed.

**Review by mfansler — CHANGES_REQUESTED** [review]

File: recipes/r-redlistr/meta.yaml (hunk 2 above)
Comment (mfansler) [reply recipes/r-redlistr/meta.yaml:56]: @Marie59 still need to add the LICENSE file to the list here.

**Comment by cbrueffer** [comment]
I think there was a misunderstanding; what Marvin meant is that the package tarball comes with a LICENSE file that you can add to the recipe by adding a `- LICENSE` entry to the `license_file` section. The `LICENSE` file you added in this PR can be removed.

**Comment by Marie59** [comment]
Indeed, I misunderstood the issue ! Hope all is well now !

**Comment by cbrueffer** [comment]
Looks good now, thanks to both of you!

**Pull request merged**